### DISCUSSION TITLE
Responding but not displaying

### DISCUSSION THREAD

**Discussion by home-apps** [FlowiseAI/Flowise#3284] [Q&A]
Hello. I read about the No Response on Initial Message issue a few months ago ([https://github.com/FlowiseAI/FlowiseChatEmbed/pull/200](https://github.com/FlowiseAI/FlowiseChatEmbed/pull/200)).

I'm not sure if it was resolved but I'm having a similar issue. I use Render and the chatbot in the render hosted dashboard works and also displays the responses, however, the same chatbot does not display responses when embedded or on shared mode.  I checked the langfuse sessions and it looks like the chatbot responds but it just does not display the responses in certain places. Anyone knows how to fix this issue or is having the same experience? Thank you

**Comment by jasonworkboost** [comment]
Same issue

**Comment by SamuelH77** [comment]
Similar issue here. I embedded my chatbot on my website using the HTML Popup option. The bot doesn't display the answers in the window when using it on the webpage. However, if I go to the chat history in Flowise, I can see that the bot was responding.
